lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -507,30 +520,114 @@ getLocationInfoAsStringImpl(const SmallPtrSetImpl<Attribute> &locationSet) {
     }
     sstr << '}';
   }
-  return sstr.str();
 }
 
-static std::string
-getLocationInfoAsStringImpl(const SmallPtrSetImpl<Attribute> &locationSet,
-                            LoweringOptions::LocationInfoStyle style) {
+/// Return the location information as a (potentially empty) string.
```

Comment:
The verb `return` is a bit strange here because there return type of this function is `void`.  Maybe something like "write location information into a stream"?